src/components/person_card/index.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import { Typography } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import './styles.css'
+const path = require('path')
+
+const useStyle = makeStyles({
+    show: {
+        cursor: "pointer",
+        fontSize: "0.8rem",
+        background: '#3f51b5',
+        padding: '0.2rem',
+        borderRadius: '5px',
+        color: 'white',
+        display: 'inline-block',
+        marginBottom: '0.2rem'
+    },
+    designation: {
+        color: '#424242',
+        fontSize: '1rem'
+    }
+});
+
+const PersonCard = (props) => {
+    const [content,setContent] = useState(props.researchArea.slice(0,100));
+    
+    const classes = useStyle();
+
+    const showLess = (e)=>{
+        setContent(props.researchArea.slice(0,100));
+    }
+
+    const showMore=(e)=>{
+        setContent(props.researchArea);
+    }
+    console.log(props.src);
+
+    return (
+        <>
+            <div className={`container`}>
+                <div className="pic">
+                    <img src={require(`../../images/${props.src}`)} alt={props.name} className="image" />
```

Comment:
Why dont we write here `/images/people/faculty` so that we have to write  image name only in JSON file ?

Reply:
Okay

Reply:
Made required changes